### DISCUSSION TITLE
allow all traffic from/to a ip address

### DISCUSSION THREAD

**Discussion by jens4topsnet** [FoobarOy/foomuuri#26] [General]
i need a rule to allow all trafffic from my server to a other server.
The iptables outgoing rule is "iptables -I OUTPUT -o etho -p all -d a.b.c.d -j ACCEPT
What is the solution with foomuuri? Must i create a service 'all udp 1-65535 ; tcp 1-65535' ?

Best Regars
--
Jens Roesiger

**Comment by kimheino** [comment]
Hi,

Above iptables-rule is:

```
localhost-public {
  oifname etho daddr a.b.c.d
  ...
}
```

If you don't care about network interface then simple `daddr a.b.c.d` line is enough. Rule defaults to "accept" and "all protocols" if not specified.

**Reply by jens4topsnet** [reply]
Thanks for the tipp. It is working.

Best Regards
